test/test_errors.py

```diff
@@ -0,0 +1,96 @@
+import os
+import shutil
+import subprocess
+from pathlib import Path
+
+import pytest
+from typer.testing import CliRunner
+
+from atopile import errors
+from atopile.cli.build import _init_python_app
+from atopile.cli.common import create_build_contexts
+
+runner = CliRunner(mix_stderr=False)
+
+PROJECT_DIR = Path("test/common/resources/test-project")
+
+
+@pytest.fixture()
+def from_project_dir(monkeypatch: pytest.MonkeyPatch):
```

Comment:
This looks like it might cause some pain at some point w/ xdist with conflicting simultaneous builds.

What about `tmp_file` instead, and copying the project to a unique tmp location for each fixture?